src/mlpack/methods/ann/init_rules/const_init.hpp

```diff
@@ -25,6 +25,14 @@ namespace ann /** Artificial Neural Network. */ {
 class ConstInitialization
 {
  public:
+
+  /**
+   *  Create the ConstantInitialization object.
+   *  Allow it to be used as a template argument.
+   */
+  ConstInitialization() : initVal(0.0) {};
```

Comment:
What about we set a default value for the existing constructor:

``ConstInitialization(const double initVal = 0) : initVal(initVal)``

Reply:
Sounds good.